app/views/user/settings.ctp

```diff
@@ -36,282 +36,318 @@ $this->set('title_for_layout', $pages->formatTitle(__('Settings', true)));
+                            <?php
+                                echo $form->hidden(
+                                    'settings.users_collections_ratings',
+                                    array(
+                                    'value' => '{{collectionRatings}}'
+                                    )
+                                );
+                            ?>
+                        </div>
+                    </md-list-item>
+                    <md-list-item>
+                        <div>
+                            <?php $nativeIndicator = $this->data['User']['settings']['native_indicator']; ?>
+                            <md-checkbox 
+                                ng-false-value="0" 
+                                ng-true-value="1"
+                                ng-model="nativeIndicator" 
+                                ng-init="nativeIndicator = <?= $nativeIndicator ?>" 
+                                class="md-primary">
+                            <label><?= __(
+                            'Display "(native)" next to username on sentences ' . 
+                            'when the owner indicated in their profile that they have a native '.
+                            'level in the language of the sentence.'
+                            ) ?></label>
+                            </md-checkbox>
+                            <?php
+                                echo $form->hidden(
+                                    'settings.native_indicator',
+                                    array(
+                                    'value' => '{{nativeIndicator}}'
+                                    )
+                                );
+                            ?>
+                        </div>
+                    </md-list-item>
+                    <md-list-item>
+                        <div>
+                        <?php $copyButton = $this->data['User']['settings']['copy_button']; ?>
+                            <md-checkbox 
+                                ng-false-value="0" 
+                                ng-true-value="1" 
+                                ng-model="copyButton" 
+                                ng-init="copyButton = <?= $copyButton ?>" 
+                                class="md-primary">
+                            <label><?= __('Display button to copy a sentence to the clipboard.') ?></label>
+                            </md-checkbox>
+                            <?php
+                                echo $form->hidden(
+                                    'settings.copy_button',
+                                    array(
+                                    'value' => '{{copyButton}}'
+                                    )
+                                );
+                            ?>
+                        </div>
+                    </md-list-item>
+                </md-list>
+            </fieldset>
+            <div layout="column">
```

Comment:
Use `<div layout="row" layout-align="center center">`. This will align the button in the center and you can remove the `save-btn` class from the md-button.